core/src/main/scala/cats/syntax/foldable.scala

```diff
@@ -46,6 +46,9 @@ final class FoldableOps[F[_], A](private val fa: F[A]) extends AnyVal {
   def foldr[B](b: Eval[B])(f: (A, Eval[B]) => Eval[B])(implicit F: Foldable[F]): Eval[B] =
     F.foldRight(fa, b)(f)
 
+  def foldA[G[_], B](implicit F: Foldable[F], ev: A <:< G[B], G: Applicative[G], B: Monoid[B]): G[B] =
+    F.foldA[G, B](fa.asInstanceOf[F[G[B]]])
```

Comment:
Simulacrum generates the same code for the syntax. I didn't figure out how to get rid of `.asInstanceOf`, because `Foldable` is invariant type. Something similar is used at `Functor#widen`.

I can make make a separate type `FoldableOps2[F[_], G[_], A]`. This way it will work without `.asInstanceOf`. Is it a better option?

Reply:
You can use `ev.apply`. `<:<[A, B]` extends `Function1[A, B]` :)

Reply:
It's impossible because I don't have access to `A`, I have only `F[A]` and `F` isn't a functor.

Reply:
Oh, duh! 👍